closed/autoconf/custom-spec.gmk.in

```diff
@@ -131,12 +131,20 @@ endif
 # Usage: $(call CodesignFile, files ...)
 ifeq (,$(CODESIGN))
   CodesignFile =
-else
+else ifeq (debug, $(MACOSX_CODESIGN_MODE))
+  CodesignFile = $(CODESIGN) --sign - \
+	--entitlements $(TOPDIR)/make/data/macosxsigning/default-debug.plist \
+	--force \
+	$1
+else ifeq (hardened, $(MACOSX_CODESIGN_MODE))
   CodesignFile = $(CODESIGN) --sign "$(MACOSX_CODESIGN_IDENTITY)" \
 	--entitlements $(TOPDIR)/make/data/macosxsigning/default.plist \
+	--force \
 	--options runtime \
```

Comment:
How come `--options runtime` and `--timestamp` aren't used in debug mode?

Reply:
This is the same as upstream, see [NativeCompilation.gmk](https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/openj9-staging/make/common/NativeCompilation.gmk#L1217).

Reply:
I suspect that in the debug mode (where `-` is used instead of `$(MACOSX_CODESIGN_IDENTITY)`), that those options aren't meaningful or are even incompatible with that use.